molecule/default/tests/test_verify_mbu_installed.py

```diff
@@ -0,0 +1,53 @@
+import os
+import pytest
+import testinfra.utils.ansible_runner
+
+testinfra_hosts = testinfra.utils.ansible_runner.AnsibleRunner(
+    os.environ['MOLECULE_INVENTORY_FILE']).get_hosts('os-infra_hosts')[:1]
+
+
+backup_folder_on_host = '/openstack/backup'
+backup_folder_on_container = '/var/backup'
+
+
+@pytest.mark.incremental
+class TestVerifyMBUInstall(object):
+    @pytest.mark.test_id('f0ef4020-a16f-11e8-97f3-fe0851898bb5')
+    @pytest.mark.jira('asc-262')
+    def test_verify_backup_folder_on_host(self, host):
+        assert host.file(backup_folder_on_host).exists
+
+    @pytest.mark.test_id('4a7ed621-a1a3-11e8-8948-6a0003552100')
+    @pytest.mark.jira('asc-262')
+    def test_veiry_backup_folder_on_container(self, host):
+        cmd1 = "ls {}".format(backup_folder_on_host)
+        output = host.run(cmd1).stdout
+        containers = output.split('\n')
+
+        # Fail the test if there is ::no container in the backup folder
+        print containers
+        assert (len(containers) > 1)
+
+        cmd2 = "test -d {}".format(backup_folder_on_container)
+        for container in containers:
+            assert (self.run_on_container(cmd2, container, host))
+
+    # Helper
+    def run_on_container(self, command, container_type, run_on_host):
```

Comment:
The `container_type` argument should be renamed to `container` since you are not looking up a type but explicitly defining the container name to connect to.